### PR TITLE
data:devWonderkind.json

### PR DESCRIPTION
**Pull request by devWonderkind**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7764"></a>

### CODE REVIEW

File: data/devWonderkind.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/devWonderkind/",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Check out my blog",
+      "url": "https://www.linkedin.com/in/devWonderkind/",
+      "icon": "SiHashnode"
+    }
+  ]
```
Comment (ChinmayMhatre) [inline data/devWonderkind.json:50]: Missing curly bracket at the end
```suggestion
  ]
}
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/devWonderkind.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/devWonderkind/",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Check out my blog",
+      "url": "https://www.linkedin.com/in/devWonderkind/",
```
Comment (ChinmayMhatre) [inline data/devWonderkind.json:47]: I guess you need to update this to a hashnode link

**Comment by ChinmayMhatre** [comment]
Hey @devWonderkind , Thank you for contributing. You have a missing bracket as mentioned in the review which is why the action is failing. Do let us know if you need help with implementing the changes.

**Comment by devWonderkind** [comment]
@ChinmayMhatre changes commit successfully.

**Review by ChinmayMhatre — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**